english/sc2/sc2.md

```diff
@@ -0,0 +1 @@
+It's 10 p.m. and you're borging, and decide to surf on google, for stacraft 1, an old game that you play some years ago, suddenly you read starcraft 2 a new version with beter graphics and you give it a chance again visiting [Starcraft 2](www.starcraft2.com)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.